packages/perseus/src/widgets/explanation.tsx

```diff
@@ -92,14 +92,16 @@ class Explanation extends React.Component<Props, State> {
         // While the button is not normally included in a block of text, it needs to be able to accommodate such a case.
         const buttonStyleOverrides = {
             height: "auto",
+            lineHeight: "inherit",
```

Comment:
Not directly related to this change, but these styles (`buttonStyleOverrides` and `labelStyle`) could (should?) go into the `const styles` object later on if there's no dynamic style calculation in them.

Reply:
(face-palm)
I over-read the WB documentation and implied something that wasn't there. Still getting used to this Stylesheet (CSS-in-JS) implementation.